main.py

```diff
@@ -1,14 +1,18 @@
 # This is a “Console calculator" for a Python AQA Bootcamp task
 import sys
+from typing import TypeVar, Callable
 
 valid_operators = ['+', '-', '/', '*']
+T = TypeVar('T', int, float)
```

Comment:
This is a nice trick, which I like the most in your version of calculator :)
But you don't have to use such advanced structures in this simple implementation. 

(this is sth definitely worth remembering)